### DISCUSSION TITLE
Plans for multinomial+ BART

### DISCUSSION THREAD

**Discussion by jlevy44** [pymc-devs/pymc3#5275] [Q&A] [answered]
Are there plans for extending BART responses to binomial, multinomial or ordinal? I would like to make the most use of the conditional mean mu for semiparametric fitting with a mix of the Particle Gibbs and HMC samplers for hierarchical modeling, but can only seem to fit on binomial and continuous outcomes.

Thanks!

**Comment by aloctavodia** [comment, marked as answer]
Hi, @jlevy44. Yes, there are plans to extend BART to work with those responses. For the moment BART has only been tested for binomial, normal and Poisson/NegativeBinomial responses.

**Comment by jlevy44** [comment]
Thank you!
